packages/app/src/app/layout.tsx

```diff
@@ -15,6 +16,7 @@ export default function RootLayout(props: PropsWithChildren) {
     <html lang='en'>
       <body>
         <Web3Provider>
+          <ToastContainer />
```

Comment:
It would be better to create a dedicated 'ToastProvider'.
That allows you to abstract the module away and standardize default functions there. 

E.g. import `react-toastify` and the css at the provider (not in the components)
```
import { toast } from 'react-toastify'
import 'react-toastify/dist/ReactToastify.css'
```

And expose a default `toast()` function from the provider which calls the react-toastify toast.
This makes it easier for future upgrades, or changing the module without going through all the components that use/import it.

Reply:
I understand, I created a context for toast, you can check it [here](https://github.com/wslyvh/nexth/pull/26/commits/77dc88dfc5987a42766549be9395007f42b730fd)